tests/api/views/test_app.py

```diff
@@ -87,10 +88,17 @@ def test_detail(client, app):
     assert set(userapp['user']) == expected_fields
 
 
-def test_delete(client, app, aws):
+@pytest.yield_fixture
+def authz():
+    with patch("controlpanel.api.cluster.auth0.AuthorizationAPI") as authz:
+        yield authz.return_value
```

Comment:
I think you could also do: `yield authz()` to have the same effect. The `return_value` is simply what you get when you call the object.

Reply:
Cheers, I'll give it a go and see if it works 👍